catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/transform/QueryListValueTransformer.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.transform;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.impl.AttributeImpl;
+import java.io.Serializable;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.boon.core.value.ValueList;
+
+public class QueryListValueTransformer implements QueryValueTransformer {
+
+  @Override
+  public boolean filter(String key, Object value) {
+    return value instanceof ValueList;
```

Comment:
❓ Why do we need to check if its specifically a boon type of list? does ValueList not implement something more generic like `List`?

Reply:
`ValueList` unfortunately does not implement `List`